packages/connectors/bookshelf/lib/models.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
         types.current = 'morphTo';
```

Comment:
the connector is getting remove I think you didn't want to rename it back to bookself :)